[DOCS-9698] Safari browser testing preview (#26732)
DataDog/documentation: content/en/synthetics/browser_tests/safari.md

3. In the **Select locations > Safari Private Locations** section, select the location named **macos-pl Safari Private Beta** that has been created for you.

  {{< img src="synthetics/browser_tests/safari/safari_private_location.jpg" alt="Screenshot of a browser test creation, showing the private locations drop down" width="70%" >}}  

4. Click on **Save & Edit Recording**, add your desired test as a subtest _playing in the main window_ : 

   {{< img src="synthetics/browser_tests/safari/safari_subtest.png" alt="Screenshot of adding an existing subtest" width="70%" >}} 

   **Note**: This references the existing test, so any modifications made to the user journey in the existing test are automatically applied to your new Safari test.

## FAQ

### Can I run a test both on Safari and non-Safari browsers?

**No**. At the moment Synthetic Monitoring browsers tests can only be run on either Safari devices **OR** non Safari devices.

### Can I run Safari tests from managed locations?

**No**. Safari browser tests are limited to a dedicated Safari private location in your account. See [list of Synthetics Private Locations][2].

> **Note**: An API key named *API Key managed by synthetics-platform* with the label `MANAGED` has been created to your org in order to create this dedicated Safari private location. **Please** **do not revoke this key** as it is required to enable Safari testing.

Reviewer: ```suggestion
**Note**: An API key named `API Key managed by synthetics-platform` with the label `MANAGED` was created and added to your org in order to create this dedicated Safari private location. **Do not revoke this key** as it is required to enable Safari testing.
```
